auth_mellon_cache.c

```diff
@@ -601,7 +601,7 @@ void am_cache_env_populate(request_rec *r, am_cache_entry_t *t)
             apr_table_set(r->subprocess_env,prefixed_varname,value);
         }
 
-        if (d->merge_env_vars != 1) {
+        if (d->merge_env_vars == NULL || *d->merge_env_vars == '\0') {
```

Comment:
I was staring at this for a while trying to see a better way to handle this that would be more intuitive, but couldn't find one. Can I suggest a small comment next to this condition describing it? E.g. something along the lines of:

> Check if merging of environment variables is disabled. This is either if it is `NULL` (default value if not configured by user) or an empty string (if specifically disabled by the user).